weasyl/define.py

```diff
@@ -166,38 +166,7 @@ def sql_number_list(target):
     elif not isinstance(target, list):
         target = [target]
 
-    return "(%s)" % (", ".join([str(i) for i in target]))
-
-
-def sql_number_series(target):
-    """
-    Returns a list of numbers suitable for placement after the SQL VALUES
-    operator in a query statement, as in "(1, 2), (3, 4), (5, 6)".
-    """
-    if not target:
-        raise ValueError
-
-    return ", ".join(sql_number_list(i) for i in target)
-
-
-def sql_string_list(target, exception=True):
-    """
-    Returns a list of strings suitable for placement after the SQL IN operator in
-    a query statement, as in "('foo', 'bar', 'baz')".
-    """
-    if not target:
-        raise ValueError
-    elif not isinstance(target, list):
-        target = [target]
-
-    return "(%s)" % (", ".join(["'%s'" % (sql_escape(i)) for i in target]))
-
-
-def sql_string_series(target):
-    if not target:
-        raise ValueError
-
-    return ", ".join(sql_string_list(i) for i in target)
+    return "(%s)" % (", ".join(["%d" % (i,) for i in target]))
```

Comment:
Nah. Incremental is good. I was just hoping we'd coincidentally obsoleted this function.